openapi-to-har.js

```diff
@@ -104,21 +104,43 @@ const getPayload = function (openApi, path, method) {
   }
 
   if (
-    openApi.paths[path][method].requestBody &&
-    openApi.paths[path][method].requestBody.content &&
-    openApi.paths[path][method].requestBody.content['application/json'] &&
-    openApi.paths[path][method].requestBody.content['application/json'].schema
+     openApi.paths[path][method].requestBody &&
+     openApi.paths[path][method].requestBody.content
   ) {
-    const sample = OpenAPISampler.sample(
-      openApi.paths[path][method].requestBody.content['application/json']
-        .schema,
-      { skipReadOnly: true },
-      openApi
-    );
-    return {
-      mimeType: 'application/json',
-      text: JSON.stringify(sample),
-    };
+    if (openApi.paths[path][method].requestBody.content['application/json'] &&
```

Comment:
What happens if both 'application/json' and `multipart/form-data` are present in requestBody.content?

/doc/api_docs/oas_components/request_bodies/upload.yml

Reply:
This is addressed here: https://github.com/bpross/openapi-snippet/pull/3